app/backend/approaches/MathTutor.py

```diff
@@ -10,8 +11,8 @@
 
 #--------------------------------------------------------------------------
 #variables needed for testing
-OPENAI_API_TYPE = "azure"
-OPENAI_API_VERSION = "2023-06-01-preview"
```

Comment:
should these be read from configuration somewhere? if they change on deployment should they change here?

Reply:
yes they are changed by the deployment. I removed them and added the " "

Reply:
oh wait I see the deployment name as well